code/game/jobs/job/medical.dm

```diff
@@ -163,8 +164,8 @@
 	name = "Pharmacist"
 	jobtype = /datum/job/pharmacist
 
-	uniform = /obj/item/clothing/under/rank/pharmacist/nt
-	suit = /obj/item/clothing/suit/storage/toggle/labcoat/pharmacist
+	uniform = /obj/item/clothing/under/rank/medical/pharmacist
```

Comment:
why is this using a default uniform when the above defaults to zenghu?

Reply:
nurse job must've slipped by me, it's removed now